app/controllers/stories_controller.rb

```diff
@@ -18,4 +18,14 @@ def show
       end
     end
   end
+
+private
+
+  def stories_template
+    "content/life-as-a-teacher/my-story-into-teaching/#{filtered_story_page}"
+  end
+
+  def filtered_story_page
+    params[:story].to_s.gsub(%r{[^a-z_\-/]}, "")
```

Comment:
Is it worth extracting this into a constant to describe what it's matching? Also, does the forward slash not need escaping and will this match a URL containing numbers? I'm thinking if they put something in like:

`stories/my-top-10-teaching-moments`

Reply:
Good call